debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/testhelper/TestInfrastructureHelper.java

```diff
@@ -27,11 +27,9 @@
 import org.testcontainers.containers.startupcheck.MinimumDurationRunningStartupCheckStrategy;
```

Comment:
` debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/testhelper/TestInfrastructureHelper.java` should be only used by REST tests and could then be entirely removed.

Reply:
Never mind. Still in use for the sink connector test pipelines.